playbooks/openstack-enroll_caasp_workers.yml

```diff
@@ -0,0 +1,19 @@
+---
+#
+# (c) Copyright 2019 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+- import_playbook: generic-enroll_caasp_nodes_in_caasp_cluster.yml
```

Comment:
This is the thing that I commented on the other commit. Why is the `openstack-X` calling the `generic-X` here? shouldnt it be all of them `openstack-X` ?

This `openstack-enroll_caasp_workers.yml` is not doing anything openstack related, only including 2 generic playbooks! That really makes me confused!

Reply:
Because this step is not really openstack specific... That action (enroll caasp workers) can probably be the same in the kvm environment, but will most likely require an extra play, which is why I think we should keep this equivalence.